docs/application/vstools-mac/README.md

```diff
@@ -0,0 +1,147 @@
+# Installing Visual Studio for Mac Extension for Tizen
+
```

Comment:
@winstone77 For now below sentence is added. If you have any suggestion to replace or add, please let me know.

Visual Studio for Mac Extension for Tizen enables you to develop Tizen aplications easily using Visual Studio Code.